browser/brave_content_browser_client_browsertest.cc

```diff
@@ -98,6 +100,11 @@ class BraveContentBrowserClientTest : public InProcessBrowserTest {
         "brave_webtorrent.html?chrome://settings");
   }
 
+  void SetUpCommandLine(base::CommandLine* command_line) override {
+    InProcessBrowserTest::SetUpCommandLine(command_line);
+    command_line->AppendSwitch("ignore-certificate-errors");
```

Comment:
please replace this with `content::ContentMockCertVerifier`.
see for details: https://bravesoftware.slack.com/archives/C7VLGSR55/p1636702893335800